faster_whisper/vad.py

```diff
@@ -323,7 +324,9 @@ def __init__(
             **inference_kwargs,
         )
 
-    def apply(self, file: AudioFile, hook: Optional[Callable] = None) -> Annotation:
+    def apply(
```

Comment:
Please change the return type in docstring:
`segmentations (Annotation): Voice activity segmentation.`